src/backend/distributed/shared_library_init.c

```diff
@@ -1896,6 +1896,7 @@ NodeConninfoGucCheckHook(char **newval, void **extra, GucSource source)
 		"sslcrl",
 		"sslmode",
 		"sslrootcert"
+		"tcp_user_timeout",
```

Comment:
I wanted to put this next to the keepalive related options, but this array needs to be order to make bsearch work.